rs/canister_sandbox/src/replica_controller/sandboxed_execution_controller.rs

```diff
@@ -1647,6 +1664,7 @@ fn evict_sandbox_processes(
     min_active_sandboxes: usize,
     max_active_sandboxes: usize,
     max_sandbox_idle_time: Duration,
+    state_reader: Arc<dyn StateReader<State = ReplicatedState>>,
```

Comment:
I think this argument could just be a reference instead of an owned `Arc`. Then you won't need to `Arc::clone` each time you call it.